databases/week2/week2_part4.sql

```diff
@@ -0,0 +1,56 @@
+
+-- -----------------------------------------------------
+-- Table Movies
+-- -----------------------------------------------------
+CREATE TABLE Movies (
+  id INT NOT NULL AUTO_INCREMENT,
```

Comment:
```suggestion
  id INT(5) NOT NULL AUTO_INCREMENT,
```
Define a specific range for the INT(n) data type.
It's consider on the performance, compression and indexes

Reply:
Hi pritalpatel
Thanks for your feedback
i have fixed all of your proposed comments